rest-app/src/test/java/com/epam/repair/rest/AddressControllerIntegrationTest.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.repair.rest;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+public class AddressControllerIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void findAll() throws Exception {
+
+        String expected = loadTestFile("address.json");
+
+        mockMvc.perform(MockMvcRequestBuilders.get("/address")
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(content().string(expected));
```

Comment:
Why as string? Why hasn't been used any JSON assertion library?

Reply:
Because the comparison file and body is in ".andExpect(content().string(expected))". If you use JSON Assert, you need to add a few lines of code. The task was to reduce the code.

Reply:
Ok. What if I add some spaces or alignment for expected JSON file? What will be with assertions in this case?

Reply:
Yes, I checked. There will be a mistake